db/compilers.xml

```diff
@@ -195,11 +195,14 @@
       <external>${PREFIX}gcc -v</external>
       <grep regexp="^[-\w]*gcc \S+ (\S+)" group="1"></grep>
     </variable>
+    <variable name="gcc_major_version">
+      <external>${PREFIX}gcc -dumpversion | cut -f1 -d.</external>
```

Comment:
This assumes that this is run using a shell and that `cut` is available, neither is guaranteed. Better to rewrite this as a grep pattern.

Reply:
Okay dokey. I'll get it changed.